File: src/IsbInterop/Tasks/Wrappers/Attachment.vb
Namespace Tasks.Wrappers

  ''' <summary>
  ''' Обертка над IAttachment.
  ''' </summary>
  Friend Class Attachment
    Inherits BaseIsbObject
    Implements IAttachment

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwIAttachment">СOM-объект IAttachment.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwIAttachment As Object, scope As IScope)
      MyBase.New(rcwIAttachment, scope)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop.CSharp/Tasks/Wrappers/Attachment.cs
namespace IsbInterop.Tasks.Wrappers
{
  /// <summary>
  /// Обертка над IAttachment.
  /// </summary>
  internal class Attachment : BaseIsbObject, IAttachment
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIAttachment">СOM-объект IAttachment.</param>
    /// <param name="scope">Область видимости.</param>
    public Attachment(object rcwIAttachment, IScope scope)
      : base(rcwIAttachment, scope) { }
  }
}
